.github/workflows/release.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-version: 18
```

Comment:
Man we still have however many repos on node 16 or earlier. Waiting  for AWS to just pull that rug out from under us.